apps/sms/test/marionette/conversation_test.js

```diff
@@ -69,6 +71,72 @@ marionette('Conversation Panel Tests', function() {
         assertIsDisplayed(messagesApp.Composer.messageInput);
       });
     });
+
+
+    suite('Long MMS thread', function() {
+      function readFile(file) {
+        return require('fs').readFileSync(file, { encoding: 'base64' });
+      }
+
+      var thread;
+      var inboxView;
+      var attachmentScreenshot = readFile(
```

Comment:
__question:__ how about moving part of this functionality to ```messages.js``` or ```views/shared/tools.js```? So it can be like:

```
messagesApp.loadAsset('mms-attachment.png'/* path relative to our base asset folder */, ....options...);
```

And maybe move reading of the file to ```suiteSetup``` since we remove the reference in ```suiteTeardown```?